sdk/cosmosdb/cosmos/README.md

```diff
@@ -254,6 +254,74 @@ for (const city of resources) {
 
 For more information on querying Cosmos DB databases using the SQL API, see [Query Azure Cosmos DB data with SQL queries][cosmos_sql_queries].
 
+### Change Feed Pull Model
+
+Change feed can be fetched for a partition key, a feed range or entire container.
```

Comment:
Change feed can be fetched for a partition key, a feed range or an entire container.